### ISSUE TITLE
Column chooser accessibility issue with tab index and incorrect label associations

### ISSUE THREAD

**Issue by yanmariomenev** [telerik/kendo-angular#4673]
### Description

In the column chooser, all hidden elements have a tab index and labels are not correctly associated with their form controls, causing accessibility issues.

### Steps To Reproduce

Open the following [demo](https://www.telerik.com/kendo-angular-ui/components/grid/accessories/column-menu#using-standalone-column-chooser) in a new window. Open the column-chooser, then run Wave and Axe accessibility tools on the page to see the errors.

### Screenshots or video

<img width="1295" height="402" alt="Image" src="https://github.com/user-attachments/assets/b0dfde18-392d-44c2-a2a0-e87fb9f53e60" />
<img width="1456" height="787" alt="Image" src="https://github.com/user-attachments/assets/89d99235-9b3a-461a-ab60-596d4bbb5334" />

**Comment by svetq** [comment]
A fix for the missing aria-label has been included in version 22.1.0-develop.3.

The rest of the accessibility issues related to the checkboxes that are a part of a list item will be handled with a separate effort as they require a more in - depth analyzis and updating the rendering of the component. You can track [this issue](https://github.com/telerik/kendo-angular/issues/4831) for further updates.
